csp-guru/datapackage.json

```diff
@@ -13,82 +13,77 @@
                         "type": "integer"
                     },
                     {
-                        "name": "Technology",
-                        "description": "distinguishes CSP technologies: Trough, Tower, Hybrid, Fresnel",
-                        "type": "string"
+                        "name": "Year_operational",
+                        "description": "Indicates the year of the first power output",
+                        "type": "integer"
                     },
                     {
-                        "name": "Capacity_MW",
-                        "description": "nominal capacity of the turbine  in MW",
-                        "type": "number"
+                        "name": "State_highes_administrative_level",
+                        "description": "Geographical description of the location where the CSP plant is located",
+                        "type": "string"
                     },
                     {
-                        "name": "Storage_capacity_h",
-                        "description": " hours the turbine can run on full load using stored heat alone in hours",
-                        "type": "number"
+                        "name": "Province_County_medium_administative_level",
+                        "description": "Geographical description of the location where the CSP plant is located",
+                        "type": "string"
                     },
                     {
-                        "name": "Field_size_m2",
-                        "description": "the mirror area ㎡",
-                        "type": "integer"
+                        "name": "City_lowest_administrative_level",
+                        "description": "Geographical description of the location where the CSP plant is located",
+                        "type": "string"
                     },
                     {
-                        "name": "Cooling",
-                        "description": "cooling technology: dry, wet, or hybrid",
+                        "name": "Location_coordinates",
+                        "description": "Location in longitude and latitude, to locate CSP plant on satelite data",
                         "type": "string"
                     },
                     {
-                        "name": "Expected_generation_GWh_per_yr",
-                        "description": "expected yearly power output GWh per year",
+                        "name": "DNI_kWh_per_m2_per_yr",
+                        "description": "Direct normal irradiance (DNI) in kWh/m²/yr, indicates the solar resource",
                         "type": "number"
                     },
                     {
-                        "name": "Load_factor",
-                        "description": "actual yearly generation divided by maximum possible yearly generation",
+                        "name": "Total_cost_million_currency_non-deflated",
```

Comment:
Are you certain this should not be `Total_cost_million_currency_non_deflated`?

Reply:
yes.